swig/core.i

```diff
@@ -51,9 +51,41 @@
 #include "server.cpp"
 #include "document.cpp"
 #include "default_constructors.cpp"
+#include <functional>
+%}
+
+%ignore getListener;
+%ignore getListeners;
+
+%inline {
+  namespace kuzzleio {
+    class NotificationListenerClass {
+      public:
+        virtual void onMessage(kuzzleio::notification_result*) = 0;
+        virtual ~NotificationListenerClass() {};
+    };
+  }
+}
+
+%extend kuzzleio::Realtime {
+  std::string subscribe(const std::string& index, const std::string& collection, const std::string& body, NotificationListenerClass* cb, room_options& options) {
```

Comment:
```suggestion
  std::string subscribe(const std::string& index, const std::string& collection, const std::string& body, NotificationListenerClass* cb, const room_options& options) {
```

Reply:
It does not change anything as this is wrapped to csharp and the const won't be there anymore in csharp. This is not meant to be used in C++